pkg/ocicni/ocicni.go

```diff
@@ -809,6 +809,13 @@ func buildCNIRuntimeConf(cacheDir string, podNetwork *PodNetwork, ifName string,
 		CapabilityArgs: map[string]interface{}{},
 	}
 
+	// Propagate existing CNI_ARGS to non-k8s consumers
+	for _, kvpairs := range strings.Split(os.Getenv("CNI_ARGS"), ";"){
+		if keyval := strings.Split(kvpairs, "="); len(keyval) == 2 {
```

Comment:
Should be `strings.SplitN(kvpairs, "=", 2)` to allow an `=` inside the environment variable body

Reply:
Done - thanks